lib/licenses.nix

```diff
@@ -1134,6 +1134,12 @@ lib.mapAttrs mkLicense ({
     fullName = "Ruby License";
   };
 
+  s3drive = {
+    spdxId = "S3Drive";
+    fullName = "S3Drive License";
+    url = "https://s3drive.app/terms";
+  };
```

Comment:
This should be `free = false` for sure.  But is it worth having a licenses.nix entry for a bespoke EULA for a single piece of software?  You could just use lib.licenses.unfree.

Reply:
Agreed!

Reply:
I wasn't sure what the protocol was for nonfree licenses - I like that suggestion, will go ahead and update.